pkg/ansible/proxy/proxy.go

```diff
@@ -32,6 +32,7 @@ import (
 	"github.com/operator-framework/operator-sdk/pkg/ansible/proxy/controllermap"
 	"github.com/operator-framework/operator-sdk/pkg/ansible/proxy/kubeconfig"
 	k8sRequest "github.com/operator-framework/operator-sdk/pkg/ansible/proxy/requestfactory"
+	osdkHandler "github.com/operator-framework/operator-sdk/pkg/handler"
```

Comment:
osdkHandler looks ugly but I'm sure gofmt will enforce it. I would prefer osdkhandler but won't hold this up for that.